.buildkite/cleanup-unused-disks.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Deletes unattached GCP disks from the "Sourcegraph Auxiliary" project
+
+set -euxo pipefail
+
+PROJECT=sourcegraph-server
+
+gcloud_command() {
+    gcloud --quiet --project="$PROJECT" "$@"
+}
+
+echo "--- Deleting unattached GCP disks from the $PROJECT project"
+
+for disk in $(gcloud_command compute disks list --filter="-users:*" --format="value(selfLink)")
```

Comment:
Does this *only* match unattached disks? When I ran this against the `sourcegraph-dev` project, it included disks like `pgsql-prod-dot-com` and `zoekt-data`, which seem very much attached and critical.

Reply:
<img width="1187" alt="Screen Shot 2019-10-16 at 4 17 56 PM" src="https://user-images.githubusercontent.com/9022011/66966014-95924b80-f030-11e9-95e5-c3b0716dbbad.png">
@beyang Those disks that you mentioned are indeed unattached.

Reply:
@beyang Additionally, ["Disks can be deleted only if they are not being used by any virtual machine instances"](https://cloud.google.com/sdk/gcloud/reference/compute/disks/delete).

Reply:
worth calling out in a comment, I think. Otherwise, LGTM.